core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -75,10 +73,11 @@ object TiUtil {
 
     if (expr.children.isEmpty) {
       expr match {
-        // bit/duration type is not allowed to be pushed down
+        // bit, duration, set, and enum type is not allowed to be pushed down
         case attr: AttributeReference if nameTypeMap.contains(attr.name) =>
           val head = nameTypeMap.get(attr.name).head
           return !head.isInstanceOf[BitType]
+          return nameTypeMap.get(attr.name).head.isPushDownSupported
```

Comment:
two `returns`?

Reply:
The second return is the real return. I'll fix it.

Reply:
The second return is the real return. I'll fix it.

Reply:
two replies and two returns... lmao